lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py

```diff
@@ -8,26 +8,28 @@
 from lldbsuite.test.lldbtest import *
 from lldbsuite.test.decorators import *
 
-class TestSwiftProgressReporting(TestBase):
 
+class TestSwiftProgressReporting(TestBase):
```

Comment:
What changed here ?

Reply:
Nothing, I didn't make any changes here. Maybe a git issue from when I created the commit?

Reply:
I'd revert these changes and see if the formatting test job fails on the PR.